config/catalog/fbc-basic-template.yaml

```
@@ -3,6 +3,10 @@ entries:
   - schema: olm.package
     name: opendatahub-operator
     defaultChannel: fast
+    icon:
+    base64data: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)